index.html

```diff
@@ -1,12 +1,23 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <title>HydraDX Omnipool - An Ocean of Liquidity</title>
     <meta charset="utf-8" />
     <link rel="icon" href="/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta name="HydraDX" content="" />
-    <title>HydraDX</title>
+    <meta name="title", content="HydraDX Omnipool - An Ocean of Liquidity" />
+    <meta name="description", content="HydraDX is a next-gen DeFi protocol which is designed to bring an ocean of liquidity to Polkadot. Our tool for the job the HydraDX Omnipool - an innovative Automated Market Maker (AMM) which unlocks unparalleled efficiencies by combining all assets in a single trading pool." />
```

Comment:
hmm, could this extra comma be the issue?

Reply:
Yeah I think so.